scripts/install_moodle.sh

```diff
@@ -29,19 +29,29 @@ function echo_error {
     echo "ERROR  - $1"
 }
 
-function downloadAndExtractMoodlePlugin {
-    local pluginTitle=$1
-    local pluginZipFileUrl=$2
-    local pluginDirPath=$3
+function get_moodle_plugin {
+    local plugin_title=$1
+    local plugin_zip_file_url=$2
+    local plugin_md5_sum=$3
+    local plugin_dir_path=$4
```

Comment:
plugin dir path changed from $3 to $4, is it intended or a typo?

Reply:
Intended. Function calls have been updated.